catalystwan/api/device_action_api.py

```diff
@@ -8,6 +8,7 @@
 
 from catalystwan.api.basic_api import DevicesAPI, DeviceStateAPI
 from catalystwan.dataclasses import Device
+from catalystwan.session import vManageBadResponseError
```

Comment:
there is no such exception in codebase

Reply:
Was this exception changed? When I tested it on testbed I was getting this exception.

Reply:
It was changed recently, here is pull request:
https://github.com/CiscoDevNet/catalystwan/pull/470